### DISCUSSION TITLE
How to adjust the prompt height

### DISCUSSION THREAD

**Discussion by ghost-ng** [prompt-toolkit/python-prompt-toolkit#1842] [Q&A] [answered]
Does anyone know how to adjust the prompt height?  This is my second application using this toolkit and I still cannot figure out how to stop the prompt from being partially up the terminal.  After taking these screenshots, I just realized maybe it has something to do with the auto-complete popup; maybe I can get it to pop above the line instead of under.

App:
https://github.com/ghost-ng/slinger

<img width="896" alt="image" src="https://github.com/prompt-toolkit/python-prompt-toolkit/assets/21272239/c61e2795-f7ce-448d-a489-2025a6481759">

<img width="569" alt="image" src="https://github.com/prompt-toolkit/python-prompt-toolkit/assets/21272239/e5a7b668-c9d6-495d-9394-b4f7d5b6c438">

<img width="416" alt="image" src="https://github.com/prompt-toolkit/python-prompt-toolkit/assets/21272239/1a4b1cf2-ec8f-4044-ad5e-5e62d98f244b">

**Comment by ghost-ng** [comment, marked as answer]
i guess all it tool was for me to realize that the extra space was because of the complete, I added this to my prompt object:

complete_while_typing=False, complete_style='readline'

<img width="379" alt="image" src="https://github.com/prompt-toolkit/python-prompt-toolkit/assets/21272239/0787ce78-9bc6-4d39-a5dc-5b043e34372a">

<img width="333" alt="image" src="https://github.com/prompt-toolkit/python-prompt-toolkit/assets/21272239/b79fc90a-bce1-4111-9c2d-d467569f14d7">

**Comment by zerocorebeta** [comment]
so you gave up on auto complete or does it still work?
